# static position definitions
# start position
start: {x: 50.2, y: 50, z: 0.0, q1: 0.0, q2: 0.0, q3: 0.0, q4: 1.0}

# base position
base: {x: 1.52, y: -0.5, z: 0.0, q1: 0.0, q2: 0.0, q3: 0.0, q4: 1.0}

doorparams: {apertureangle: 0, walldistance: 2, mindoorwidth: 0.5}

endPose: {x: 0.7, y: 0, w: 3.14}

